packages/x-adapter/README.md

```diff
@@ -1,13 +1,224 @@
+
+You can check the
+[x-platform-adapter](https://github.com/empathyco/x/tree/main/packages/x-adapter-platform) library.
+You will find a sample implementation of the `x-adapter` library based on the
+[Search Platform API](https://docs.empathy.co/develop-empathy-platform/api-reference/search-api.html),
+and also some guidance on how to extend it for your needs.
+
+## Test
+
+```
+npm test
+```
+
+## Changelog
+
+[Changelog summary](https://github.com/empathyco/x/blob/main/packages/x-adapter/CHANGELOG.md)
+
+## Contributors
+
+|    <a href="https://github.com/tajespasarela"><img src="https://avatars.githubusercontent.com/u/5759712?v=4" width="100px;" alt=""/><br /><br /><b>Iván Tajes</b></a>    | <a href="https://github.com/javieri-empathy"><img src="https://avatars.githubusercontent.com/u/68222542?v=4" width="100px;" alt=""/><br /><br /><b>Javier Iglesias</b></a> |   <a href="https://github.com/LuisMartinez15"><img src="https://avatars.githubusercontent.com/u/6247440?v=4" width="100px;" alt=""/><br /><br /><b>Luís Martínez</b></a>   |  <a href="https://github.com/tiborux"><img src="https://avatars.githubusercontent.com/u/6597815?v=4" width="100px;" alt=""/><br /><br /><b>Beltrán García</b></a>   |
+| :----------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| <a href="https://github.com/CachedaCodes"><img src="https://avatars.githubusercontent.com/u/7124620?v=4" width="100px;" alt=""/><br /><br /><b>Guillermo Cacheda</b></a> |      <a href="https://github.com/diegopf"><img src="https://avatars.githubusercontent.com/u/7504736?v=4" width="100px;" alt=""/><br /><br /><b>Diego Pascual</b></a>       | <a href="https://github.com/mnavarroespinosa"><img src="https://avatars.githubusercontent.com/u/77450928?v=4" width="100px;" alt=""/><br /><br /><b>Manuel Navarro</b></a> | <a href="https://github.com/ajperezbau"><img src="https://avatars.githubusercontent.com/u/75546736?v=4" width="100px;" alt=""/><br /><br /><b>Abraham Pérez</b></a> |
+
+## Contributing
+
+To start contributing to the project, please take a look to
+our **[Contributing Guide](https://github.com/empathyco/x/blob/HEAD/.github/CONTRIBUTING.md).**
+
+## License
 
-To start contributing to the project, please take a look to our
-[Contributing Guide](../../.github/CONTRIBUTING.md).
+[empathyco/x License](https://github.com/empathyco/x/blob/main/LICENSE)
```

Comment:
We should put the license file in every package. Could you add it to this package? You can copy the same file that we have in x-components package: https://github.com/empathyco/x/blob/main/packages/x-components/LICENSE

Then, in this README, link it to that new file.